src/components/PostList/PostList.tsx

```diff
@@ -1,3 +1,19 @@
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import React from 'react';
+import './PostList.scss';
+
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../types/Post';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <>
```

Comment:
you don't need react fragment here (and in all the homework) 😉

Reply:
I did not understand this comment.
Do I need to delete React.FC?

Reply:
no, you just need to delete fragment <> </>